microstoragetest/storage.go

```diff
@@ -0,0 +1,18 @@
+package microstoragetest
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/microstorage"
+	"github.com/giantswarm/microstorage/memory"
+)
+
+// Must creates a configured storage ready to be used in tests.
+func Must() microstorage.Storage {
```

Comment:
Ok, I am not interested in this anymore. Name it how you think it is right. Sorry for stressing on this unimportant detail.